pipelines/hubspot/__init__.py

```diff
@@ -0,0 +1,68 @@
+from typing import Sequence, Iterator
+
+import dlt
+
+from dlt.common.typing import TDataItems
+from dlt.extract.source import DltResource
+
+from pipelines.hubspot.client import fetch_data
```

Comment:
please use relative imports ie.
`from .hubspot.client import fetch_data`

we'll need it to properly distribute code.
https://github.com/dlt-hub/pipelines#python-module-import-structure